cpp/doxygen/rapids.css

```diff
@@ -0,0 +1,81 @@
+
```

Comment:
Do you know if this file needs a copyright heading?

Reply:
I guess technically yes but it's not a big deal. Will add it when I fix the tab.